src/components/Input/Input.tsx

```diff
@@ -163,31 +158,14 @@ class Input extends AutoControlledComponent<any, any> {
   }
 
   renderComponent({ ElementType, classes, rest, styles }) {
-    const { children, clearable, input, type } = this.props
+    const { type } = this.props
     const [htmlInputProps, restProps] = this.partitionProps()
 
     const inputClasses = classes.input
-    const iconClasses = classes.icon
-
-    // Render with children
-    // ----------------------------------------
-    if (childrenExist(children)) {
-      // add htmlInputProps to the `<input />` child
-      const childElements = _.map(React.Children.toArray(children), child => {
-        if (child.type !== 'input') return child
-        return React.cloneElement(child, this.handleChildOverrides(child, htmlInputProps))
-      })
-
-      return (
-        <ElementType {...rest} className={classes.root}>
-          {childElements}
-        </ElementType>
-      )
-    }
 
     return (
       <ElementType {...rest} className={classes.root} {...htmlInputProps}>
-        {createHTMLInput(input || type, {
+        {createHTMLInput(type || 'text', {
```

Comment:
The defaultValue of `type` should be `text` instead.  Otherwise, this default value behavior is not going to show up in the documentation.